ambry-clustermap/src/test/java/com.github.ambry.clustermap/ClusterMapUtilsTest.java

```diff
@@ -26,6 +25,7 @@
 import org.junit.Test;
 import org.mockito.Mockito;
 
+import static com.github.ambry.utils.TestUtils.*;
```

Comment:
Why static import in this class?

Reply:
For the tests in the clustermap package, there is another class called `com.github.ambry.clustermap.TestUtils`. To avoid having to write out the fully qualified classname every time getRandomString is used, I did a static import. I suppose the best thing to do would be to change the other package's name to `ClustermapTestUtils`, but I didn't want to do that in the same PR to prevent this one from getting even bigger.